css/index.css

```diff
@@ -1,3 +1,407 @@
+  margin-right: 0;
+}
+
+.main-content .middle-img {
+  margin-top: 45px;
+}
+
+.main-content .bottom-content {
+  margin-top: 37px;
+  padding: 0 10px;
+}
+
+.main-content .bottom-content .text-content {
+  width: 275px;
+  display: inline-block;
+  margin-right: 12px;
+}
+
+.main-content .bottom-content .text-content:last-child {
+  margin-right: 0;
+}
+
+.contact {
+  margin-top: 57px;
+  border-top: 1px solid black;
+  padding: 39px 10px 0;
+}
+
+.contact p {
+  margin: 20px 0;
+}
+
+footer {
+  margin: 32px 0;
+  text-align: center;
+  font-size: 14px;
+}
+@media (max-width: 800px){
+    .container{
+    width: 700px;
+    margin: 0 auto; 
+    }
+    header{
+        height: 150px;
+    }
+    header .logo {
+        display: inline-block;
+        top: 50%;
+        transform: translateY(-200%);
+        margin:30px 0 0 250px;
+        padding: 8px;
+      }
+      header nav {
+        display: inline-block;
+        margin-bottom: 0;
+        margin-top : 10px;
+        width: 690px;
+        height: 80px;
+        padding: 10px;
+      }
+    header nav a {
+        display: inline-block;
+        margin: 50px 0 0 10px;
+        font-size: 15px;
+        color: gray;
+        text-decoration: none;
+        padding: 10px;
+        margin:60px 10px 0 40px;
+      }
+      header nav a:hover {
+        text-decoration: underline;
+      }
+
+      .cta .cta-text {
+        display: inline-block;  
+        margin-left: 160px;
+        margin-bottom: 0px;
+        margin-top: 0; 
+        text-align: center;
+      }
+      .cta .cta-text h1{
+          font-size: 50px;
+          margin-top:0 ;
+      }
+      .cta .cta-text button {
+        background: white;
+        font-size: 9px;
+        border: 1px solid black;
+        cursor: pointer;
+
+      }
+      .cta{
+          margin-bottom:0;
+          height: 160px;
+      }
+      .cta img {
+        display: none;
+      }
+      .main-content .middle-img {
+        margin-top: 45px;
+        width: 700px;
+      }
+      .main-content {
+        border-top: 1px solid black;
+        margin-top: 76px;
+        padding-top: 39px;
+      }
+      
+      .main-content .top-content {
+        padding: 0 5px;
+      }
+      
+      .main-content .top-content .text-content {
+        width: 330px;
+        display: inline-block;
+        margin-right: 12px;
+      }
+      
+      .main-content .top-content .text-content:last-child {
+        margin-right: 0;
+      }
+      
+      .main-content .middle-img {
+        margin-top: 45px;
+      }
+      
+      .main-content .bottom-content {
+        margin-top: 37px;
+        padding: 0 10px;
+      }
+      
+      .main-content .bottom-content .text-content {
+        width: 220px;
+        display: inline-block;
+        margin-right: 6px;
+      }
+      
+      .main-content .bottom-content .text-content:last-child {
+        margin-right: 0;
+      }
+
+      @media (max-width: 500px){
+        .container{
```

Comment:
You set some height on your header in your media query for 800 which is impacting the look of the nav for the 500px. It looks good but could be more centered.